tests/conftest.py

```diff
@@ -201,6 +201,13 @@ def is_package_url_specified(request):
         pytest.skip('Test requires a url for a dev package.')
 
 
+@pytest.fixture(scope='session')
+def min_product_version(request):
+    if request.param > request.config.getoption('--product_version'):
```

Comment:
> or = (should be {request.param} or newer) it will not work in case 2021.2 and you have 'min_product_version', ['2021.2']

Reply:
It works in case '2021.2' because '2021.2' > '2021.2' = false